ui/app/AppLayouts/Browser/popups/BrowserWalletMenu.qml

```diff
@@ -204,10 +204,12 @@ Popup {
 
             AssetsView {
                 id: assetsTab
+                locale: RootStore.locale
```

Comment:
I don't know... but we don't really have to pass the `locale` around for every component; it's the same thing for the whole app. Seems like overkill to me :)